### ISSUE TITLE
base training configuration

### ISSUE THREAD

**Issue by crankyz** [dr-pato/audio_visual_speech_enhancement#2]
Hello, Giovanni. Could you provide the best configuration for training grid database. Thank you.

**Comment by dr-pato** [comment]
Hi crankyz,
which of the models are you interested in? Note that some of iperparameters are reported in the paper.

**Comment by crankyz** [comment]
AV c-ref, as the best results at average.

**Comment by dr-pato** [comment]
If you want to train the AV c-ref you have to do the following steps (described in the paper in section 3.2):
1. Train the "vl2m" model (adam optimizer with 10^-4 as initial learning rate).
2. Save estimated TBMs of training, validation and test set with "testing" subcommand.
3. Train "av_concat_mask_ref" with oracle TBMs (adam optimizer with 10^-3 as initial learning rate).
4. Generate TFRecords using the estimated TBMs instead of oracle TBMs.
5. Select the best epoch of trained "av_concat_mask_ref" model and fine-tune the model with TFRecords of point 4.

Let me know if you are able to do all steps correctly.
